tests/resources/nimbus-testbeds/vic-nfs.rb

```diff
@@ -0,0 +1,58 @@
+#This file is needed because there are 3 functions in it which are used in this testbed.
+require '/mts/git/nimbus/lib/testframeworks/testng/testng.rb'
+oneGB = 1 * 1000 * 1000
+$testbed = Proc.new do |sharedStorageStyle, esxStyle|
+  esxStyle ||= 'fullInstall'
+  esxStyle = esxStyle.to_s
+  sharedStorageStyle = sharedStorageStyle.to_s
+
+  testbed = {
+    'version' => 3,
+    'name' => "vsan-1esx-allflash-#{sharedStorageStyle}-#{esxStyle}",
+    'esx' => (0..0).map do |i|
+      {
+        'name' => "esx.#{i}",
+        'style' => esxStyle,
+        'numMem' => 64 * 1024,
+        'numCPUs' => 6,
+        'disks' => [ 250 * oneGB ],
+        'disableNfsMounts' => true,
+        'ssds' => [ 200 * oneGB, 250 * oneGB ],
+        'nics' => 2,
+        'staf' => false,
+        'desiredPassword' => 'ca$hc0w',
```

Comment:
e2eFunctionalTest

Reply:
I am updating the password to e2eFunctionalTest after the ESX is created. https://github.com/rajanashok/vic/blob/15813ff2ec286414a36201f381079212ca9b55bc/tests/resources/Nimbus-Util.robot#L445